### DISCUSSION TITLE
Possibilité de retirer le manteau (Ou le haut sans retirer le maillot de corps)

### DISCUSSION THREAD

**Discussion by Jake-Freaks** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#270] [Ideas]
Coucou. Ce serait imaginable de faire un truc pour qu'on puisse dévêtir qu'une partie du haut quand on le souhaite dans la liste de "gestion de la tenue".

Je suis pas codeur donc je glisse juste l'idée.

**Comment by Laikker** [comment]
Compliqué car les maillots de corps sont souvent tronqué pour affiché que le nécessaire, genre https://wiki.rage.mp/index.php?title=File:Clothing_M_8_3.jpg, et donc ne peuvent pas s'utiliser seul
